docs/running/debugging.rst

```diff
@@ -5,8 +5,73 @@ Toil Debugging
 
 Toil has a number of tools to assist in debugging.  Here we provide help in working through potential problems that a user might encounter in attempting to run a workflow.
 
-Introspecting the Jobstore
---------------------------
+Reading the Log
+---------------
+
+Usually, at the end of a failed Toil worklfow, Toil will reproduce the job logs for the jobs that failed. You can look at the end of your workflow log and use the job logs to identify which jobs are failing and why.
+
+Finding Failed Jobs in the Jobstore 
+-----------------------------------
+
+The ``toil status`` command (:ref:`cli_status`) can be used with the ``--failed`` option to list all failed jobs in a Toil job store.
+
+You can also use it with the ``--logs`` option to retrieve per-job logs from the job store, for failed jobs that left them. These logs might be useful for diagnosing and fixing the problem.
+
+Running a Job Locally
+---------------------
+
+If you have a failing job's ID or name, you can reproduce its failure on your local machine with ``toil debug-job``. See :ref:`cli_debug_job`.
+
+For example, say you have this WDL workflow in ``test.wdl``. This workflow **cannot succeed**, due to the typo in the echo command::
```

Comment:
Is this also working for CWL?

Reply:
Yeah. I think in CWL we put the dotted name in as the "unit" name and the "display" name also has the class and a space:

https://github.com/DataBiosphere/toil/blob/6770622c8a380a8b0040646436c343a7af71f9c9/src/toil/cwl/cwltoil.py#L2089C9-L2089C18

But the lookup also matches on unit name so I think it should work.